tests/suites/test_suite_rsa.function

```diff
@@ -21,6 +21,9 @@ void rsa_invalid_param()
 
     mbedtls_rsa_init(&ctx);
 
+    TEST_EQUAL(mbedtls_rsa_get_padding_mode(&ctx), MBEDTLS_RSA_PKCS_V15);
+    TEST_EQUAL(mbedtls_rsa_get_hash_id(&ctx), MBEDTLS_MD_NONE);
```

Comment:
We need more testing than that: we need positive testing, to ensure this is actually returning the value in the context. We need to test that after `mbedtls_rsa_set_padding(ctx, p, h)`, `get_padding_mode` returns `p` and `get_hash_id` returns `h`. Tried for different values of `p` and `h`, not just the defaults.

Reply:
I didn't find a area to verify `padding mode` and `hash_id` so previously I added this [function](https://github.com/Mbed-TLS/mbedtls/pull/7190/commits/097147540d7b0fb4526d63334a6c2fd6beee35b9#diff-c50842e3873630fc85090fd07b82284b4b3021a436b27e88bebf413ffefa7f65R174) to `mbedtls_rsa_set_padding` first then verify `padding_mode` and `hash_id`. Does that look good to you?

Reply:
`test_suite_rsa` is about RSA key management which doesn't care about those values, so you'll find the calls to `mbedtls_rsa_set_padding` in the test suites that cover RSA operations: `test_suite_pkcs1_v15.function` and `test_suite_pkcs1_v21.function`. Please add a check of the get functions in one or more places after a call to `mbedtls_rsa_set_padding` in each of these suites.